### PR TITLE
Feature/selection contains entity

### PR DESCRIPTION
**Pull request by johanneslumpe**
Depends on and thus contains https://github.com/draft-js-utils/draft-js-utils/pull/5

### CODE REVIEW

**Comment by sstur** [comment]
Thanks @johanneslumpe. Any tests for this would be welcome : )

Side note: I hate how in Github, pullrequests map to branches that must be diffed against a branch in the upstream repo, therefor making all features that depend on another feature include all the code for both! Is there any way around this? I believe what I'm looking for is called "stacked diffs" in Phabricator but not sure that maps to anything in Github world.

**Comment by johanneslumpe** [comment]
@sstur Also here, will add tests soon!

I'm not sure if there is a way around that :( I just manually included the PR because it wouldn't be functional on its own without that code.

**Comment by jaaberg** [comment]
@sstur Unfortunately, I don't think that is a thing in Github 

If we are being picky, the variable may be named entityFound instead of entitiesFound. 

Otherwise it looks good :)

**Comment by johanneslumpe** [comment]
@jaaberg fixed variable name @sstur here are the tests

File: src/__tests__/getSelectedBlocks-test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    'split-block',
+  );
+
+  return EditorState.forceSelection(
+    newEditorState,
+    newContentState.getSelectionAfter(),
+  );
+};
+
+const editorState = [...Array(3)].reduce((state) => splitLastBlock(state), EditorState.createEmpty());
```
Comment (sstur) [inline src/__tests__/getSelectedBlocks-test.js:25]: Nice use of spread operator!

**Comment by sstur** [comment]
Thanks @johanneslumpe! I'll sign off on this. Merge at your convenience.

**Pull request merged**